payments/management/commands/one_time_sms_message.py

```diff
@@ -0,0 +1,171 @@
+            )
+            .values("id")
+        )
+        orders = Order.objects.filter(
+            Q(
+                # Valid leases
+                Q(_lease_object_id__in=leases)
+                # Invoices that haven't been paid
+                & Q(status=OrderStatus.WAITING)
+                & Q(
+                    # Either "normal" customers
+                    Q(customer__organization__isnull=True)
+                    # Or "billable" organization customers
+                    | Q(
+                        customer__organization__organization_type__in=(
+                            OrganizationType.COMPANY,
+                            OrganizationType.OTHER,
+                        )
+                    )
+                )
+                # Only for production, since no invoices will have phone, it will work
+                # as a flag to check which ones to send
+                # customer_phone__isnull=True,
+            )
+        )
+
+        profile_service = ProfileService(profile_token=profile_token)
+        sms_service = SMSNotificationService()
+        payment_provider = BamboraPayformProvider(
+            ui_return_url=settings.VENE_UI_RETURN_URL
+        )
+
+        MESSAGE = (
+            "Hyvä asiakas. Tässä maksulinkki venepaikan maksamiseen purjehduskaudelle 2021. "
+            "Maksathan laskun 31.1.2021 mennessä.\n\n"
+            "Dear Customer. Here is a payment link for paying for a berth for the sailing season 2021. "
+            "Please pay the invoice by January 31, 2021.\n\n"
+            "{url}"
+        )
+
+        if dry_run:
+            invoices = orders.count()
+            without_phone = orders.filter(
+                Q(Q(customer_phone__isnull=True) | Q(customer_phone=""))
+            ).count()
+
+            self.stdout.write(f"Invoices to be sent: {invoices}")
+            self.stdout.write(f"Customers without phone: {without_phone}")
+
+            return
+
+        for order in orders:
+            # If the order doesn't have a customer phone number, fetch it from Profiili
+            if not order.customer_phone:
+                logger.debug(f"Fetching profile for: {order.customer.id}")
+                customer = profile_service.get_profile(order.customer.id)
+
+                # If the customer doesn't have a phone in Profiili, add it to the failures
+                # so the admins can handle it manually
+                if not customer.phone:
+                    no_customer_phone.append(f"{order.id};{order.customer.id}\n")
+                    continue
+
+                # Save the phone to the order, in case it is required later
+                order.customer_phone = customer.phone
+                order.save()
+            else:
+                logger.debug(f"Phone found for: {order.customer.id}")
+
+            message = MESSAGE.format(
+                url=payment_provider.get_payment_email_url(order, "fi")
+            )
```

Comment:
Do we need two urls in the message, "fi" and "en"?

Reply:
I don't think so.